packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -97,6 +101,25 @@ contract OPStackManager is ISemver {
         address resolvedDelegateProxy;
     }
 
+    /// TODO: Design question, do we still want this setRelease functionality if we have initialize functionality?
```

Comment:
I'd say let's keep this as simple as possible, and delete `setRelease` and just use the initializer to do this

Reply:
Done in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23